src/NuGetPackageVerifier/Program.cs

```diff
@@ -25,6 +25,13 @@ public static int Main(string[] args)
 
             // TODO: Get this from the command line
             var ignoreAssistanceMode = IgnoreAssistanceMode.ShowAll;
+            var hideInfoLogs = false;
+
+            if (args.Length > 0 && string.Equals("--quiet", args[0], StringComparison.OrdinalIgnoreCase))
```

Comment:
> do we want to make the default Normal and hide the random spew we get today Info?

Does this mean we'll have four log levels instead of three?

> can we just make this --log-level?

I personally feel having a min log level switch is unnecessary at this point and we can add it later if we need it. But I don't feel strongly about this either way. Let me know if you think it is necessary.